src/Core/Rexl.Bind/Operations/BuiltinFunctions.cs

```diff
@@ -378,6 +378,9 @@ private BuiltinFunctions()
         AddOne(TextTrimFunc.TrimStart, new Sig(S.AboutTrimStart, A.Create(S.ArgSource, S.AboutTrimStart_Source)));
         AddOne(TextTrimFunc.TrimEnd, new Sig(S.AboutTrimEnd, A.Create(S.ArgSource, S.AboutTrimEnd_Source)));
 
+        AddOne(TextPadFunc.Left, new Sig(S.AboutPadLeft, A.Create(S.ArgSource, S.AboutPad_Source)));
```

Comment:
We should use the terms "Start" and "End" instead of "Left" and "Right". In a right to left writing system, "Left" and "Right" will be backwards but "Start" and "End" will still be correct. Note that "Trim" uses this.

Reply:
@shonk-msft My thought process is the "start" of an English sentence would be to the left but the "start" of an Arabic sentence would be to the right, so "left" and "right" are more consistent across languages than "start" and "end." Does C# PadLeft/PadRight adjust to different writing systems like that? I couldn't find it in the docs.

Reply:
By "left", the .Net API means _smaller character indices_. That is, .Net's `string.PadLeft` will add characters with smaller indices. When the active display script is right to left, these character positions will naturally be displayed to the right of the original characters. The .Net API doesn't have any "display script" information available to it.

Generally, we shouldn't use "left" and "right" in the text function names. If you find any, we should fix them. We can use "start", "end", "first", "last", but not "left" and "right".